packages/grafana-prometheus/src/utf8_support.ts

```diff
@@ -0,0 +1,81 @@
+export const utf8Support = (value: string) => {
```

Comment:
I feel that this function could be named more clearly. I don't have a clear idea about what it should be, but it should describe the action that it performs. Maybe something like `convertToUtf8`, `ensureUtf8Support`, or `utf8Format`.

Reply:
I don't like the name either :) But it was done in the main big PR which I closed to split it up into smaller PRs. As I am splitting up, to prevent issues, I kept the names as they are. After all PRs merged I'll rename things.